src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -43,25 +85,31 @@ def validate_request_update_args(request_args, config, reqmgr_db_service, param)
     """
     # this needs to be deleted for validation
     request_name = request_args.pop("RequestName")
+
     couchurl = '%s/%s' % (config.couch_host, config.couch_reqmgr_db)
     workload = WMWorkloadHelper()
     workload.loadSpecFromCouch(couchurl, request_name)
 
+    request = reqmgr_db_service.getRequestByNames(request_name)
```

Comment:
Just a note, we can inspect the `workload` object above to see if it provides the request dictionary/json. If it does, then we don't need this extra call.

Reply:
I wasted a lot of time .... inspecting it... and believe I found it not. If it is there, then it must be on a place to be considered far from intuitive.